scripts/DeployFraxOFTProtocol/mfrax-aptos-move.layerzero.config.ts

```diff
@@ -23,15 +25,13 @@ type lzConfigType = {
     sendLib302: string
 }
 
+const dvnConfigPath = "config/dvn"
+const zeroBytes32 = '0x0000000000000000000000000000000000000000000000000000000000000000'
+
 const chainIds = [
```

Comment:
Will this need to be updated when we deploy to new chains or no?  Which chains do we expect to wire aptos to long-term?

Reply:
This needs to update when we would like to new chains.